src/components/DeckBuilderDryRunView/index.js

```diff
@@ -224,7 +224,7 @@ class DeckBuilderDryRunView extends React.Component {
   containsFreeze = deck => {
     const deckIds = deck.map(card => card.id)
     const freezeCards = ['W1', 'W2', 'W4', 'W6', 'W8', 'W11']
-    return deckIds.some(id => freezeCards.includes(id))
+    return deckIds.some(id => freezeCards.includes(id.split('#')[0]))
```

Comment:
I can see how this is going to be absolute hell quite quickly; I’m not sure I want us to have to split the ID everywhere we want to use it. Maybe we should handle things a little differently. Perhaps we could have 2 IDs per card: `id`, which is the card ID like right now, and `idx` which is a unique identifier so we can differentiate `W1` from `W1` when necessary. This way, all the components relying on `id` (like `Deck`) still work out of the box, we can differentiate cards within the dry-runner with their `idx`. What do you think?